apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/RetryTest.cs

```diff
@@ -81,14 +79,13 @@ public void ExceptionAfterRetryExhaustedTest()
 
         AssertAttempts(
            retryOptions: retryOptions,
-           statusCode: 502,
-           numOfFailures: 4,
+           requestStatusCodes: new[] { 502, 502, 502, 502, 200 },
            maximumRetries: 3,
            success: false,
            expectedBackOffs: expectedBackOffs);
     }
 
-    private static void AssertAttempts(RetryOptions retryOptions, int statusCode, int numOfFailures, int maximumRetries, bool success, params int[] expectedBackOffs)
+    private static void AssertAttempts(RetryOptions retryOptions, int[] requestStatusCodes, int maximumRetries, bool success, params int[] expectedBackOffs)
```

Comment:
Rename to responseStatusCodes (as that's where the status codes are reported).

Reply:
Do we actually need `maximumRetries`? That's not part of RetryHandler, so I think it would be best to remove it for now.